crates/store/src/state_db.rs

```diff
@@ -86,6 +116,102 @@ impl StateDBTransaction {
         }
         Ok(())
     }
+
+    fn get_block_num_and_tx_index(
+        inner: &StoreTransaction,
```

Comment:
```suggestion
        db: &StoreTransaction,
```

The name `inner` doesn't make sense here